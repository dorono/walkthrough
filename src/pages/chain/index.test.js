import React from 'react';
import {shallow} from 'enzyme';
import {ChainPage} from './index';

const defaultProps = {
    data: {
        external_ids: ['UlNBLVNIQTUxMg=='],
    },
    location: {},
};

describe('Chain', () => {
    it('should render without errors', () => {
        shallow(<ChainPage {...defaultProps} />);
    });

    it('should match snapshot', () => {
        expect(shallow(<ChainPage {...defaultProps} />)).toMatchSnapshot();
    });
});
