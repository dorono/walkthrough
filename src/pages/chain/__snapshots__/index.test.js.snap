// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chain should match snapshot 1`] = `
<div>
  <Container
    fullWidthBannerText="This chain has not yet been written to the Factom Blockchain."
    primary={true}
    showFullWidthBanner={false}
    title="Chain"
  >
    <Vertical>
      <Box
        type="outline"
      >
        <Label>
          IMMUTABILITY STAGE
        </Label>
        <span
          className="capitalized"
        />
      </Box>
      <Box
        type="outline"
      >
        <Label>
          CHAIN ID
        </Label>
        <withRouter(Hash)
          type="chain"
        />
      </Box>
      <Box>
        <Label>
          EXTERNAL IDS
        </Label>
        <ExternalIdList
          className="multiline"
          externalIds={
            Array [
              "UlNBLVNIQTUxMg==",
            ]
          }
        />
      </Box>
    </Vertical>
  </Container>
  <DataLoader
    entriesUrl="/chains/undefined/entries"
    renderContent={[Function]}
  />
</div>
`;
